rm-datasource/src/main/java/io/seata/rm/datasource/exec/DeleteExecutor.java

```diff
@@ -71,6 +71,7 @@ private String buildBeforeImageSQL(SQLDeleteRecognizer visitor, TableMeta tableM
         suffix.append(" FOR UPDATE");
         StringJoiner selectSQLAppender = new StringJoiner(", ", "SELECT ", suffix.toString());
         for (String column : tableMeta.getAllColumns().keySet()) {
+            column = "`" + column + "`";
```

Comment:
why add mysql escape? other database how to do?

Reply:
it fixes the issue #2662 , the solution works in MySQL, maybe does not work in other databases.

Reply:
How about remove this code? because this is the other problem, you can use a new pr to fix it.